app/search/classification/tests/test_domain.py

```diff
@@ -0,0 +1,45 @@
+from unittest import TestCase
+from urllib.parse import ParseResult
+
+from nose.tools import eq_, ok_
+
+from app.search.classification.domain import DomainClassifier
+
+
+DOMAIN = 'www.mozilla.com'
+
+
+class TestDomainClassifier(TestCase):
+    def _result(self, url):
+        return {
+            'url': url
+        }
+
+    def _classifier(self, url):
+        return DomainClassifier(self._result(url))
+
+    def _matches(self, url):
+        return self._classifier(url).is_match(self._result(url))
+
+    def _enhance(self, url):
+        return self._classifier(url).enhance()
+
+    def test_init(self):
```

Comment:
Not sure I understand the intent / value of this test. Is this just to be sure the thing works from start to finish?